src/map_reads.cpp

```diff
@@ -106,20 +109,21 @@ struct MapAndPhase
             synchro->lock ();
 
             nbOfReadsProcessed+=NB_OF_READS_NOTIFICATION_MAP_AND_PHASE;
-            cerr << '\r' << nbOfReadsProcessed << " reads processed.";
```

Comment:
Can we keep the progress messages on stderr, and leave the final counts on cout

Reply:
Sure, if that's where you prefer your output. It just makes it a bit confusing for those who want to redirect error messages and regular output to different log files.

Reply:
I tend to write everything to stderr, maybe I should follow that here and also write the count output there too. I agree it is confusing when you set a pipe and it doesn't capture all the messages